universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -102,14 +102,15 @@ public PresentsErrors errorsPresenter() {
     switch (getType()) {
       case ADDRESS:
         return getAddressQuestion();
-      case DROPDOWN:
-        return getSingleSelectQuestion();
       case NAME:
         return getNameQuestion();
       case NUMBER:
         return getNumberQuestion();
       case TEXT:
         return getTextQuestion();
+      case DROPDOWN:
+      case RADIO_BUTTON:
+        return getSingleSelectQuestion();
```

Comment:
Does dropdown not have its own renderer?

Reply:
Good question! It has its own renderer, but both radio and dropdown questions share the same `SingleSelectQuestion` applicant question type (handles logic for getting the single string selection, getting the answer option strings, etc.)